test/test_roscpp/test/CMakeLists.txt

```diff
@@ -1,3 +1,5 @@
+SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -pthread")
```

Comment:
Is that also necessary for other ROS packages which use this modified version of `roscpp`?